benches/README.md

```diff
@@ -7,17 +7,57 @@ By convention, Criterion benchmarks have prefix `ct`, iai benchmark names start
 To execute a benchmark, run
 
 ```bash
-cargo bench -F test-fixture --bench <benchmark_name>
+cargo bench -F enable-benches --bench <benchmark_name>
```

Comment:
I find it annoying that `cargo bench --bench foo` doesn't change the build configuration so that the necessary features are turned on.

Reply:
yea, I also wish I could specify default features for profiles.